package/network/config/ltq-vdsl-app/files/10_atm_ptm.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
```

Comment:
> Got your point but still NAK from me. Other devs might have a different opinion on this. I'm the opinion the way to go is getting the dsl subsystem reset working. Everything else is a dirty hack. Sorry.

I can understand you, but I need an implementation where the cpe can recover itself in any situation. So this feature will land on my private patch stack. :-)